tests/gsrpc/extrinsics/dataSubmission/dataSubmission.go

```diff
@@ -114,7 +115,7 @@ func main() {
 		panic(err)
 	}
 	size := 0
-	if config.Size == 0  || config.Size > -1 {
+	if config.Size <= 0 {
 		size = 100
 	}
```

Comment:
This is probably a bug, since `config.Size` is ignored, we should probably repeat same code from `dataSubmitAndWatch.go`, or, if possible, assign default value to `config.Size` and just use it.